### ISSUE TITLE
Not an issue - more of a doubt

### ISSUE THREAD

**Issue by dakinianaya** [PIut02/ROG-Zephyrus-G14-GA401-Hackintosh#23]
Is there a specific bios version we need to be in before performing the installation? It has not mentioned anywhere in this repo but I read it somewhere in 2022 model guide. I have 2021 G14 with 4900HS RTX 2060 Max Q. Hope there shouldn't be any change needed for 4900HS. @PIut02 Please guide. I badly need xcode in my machine.

**Comment by PIut02** [comment]
There is no need for a specific BIOS version, I tried it on the latest BIOS with no problem, and the other one using the 4900HS is 2020 version.

**Comment by dakinianaya** [comment]
Hi @PIut02 I am able to boot till recovery screen with 2020 EFI but unable to install due to no internet. I have Intel wifi AX 200. But still not working. I even tried to connect my mobile with usb tethering. Didn't work. Any ideas or suggestions?

**Comment by PIut02** [comment]
Ventura requires the use of a specific branch: https://github.com/PIut02/ROG-Zephyrus-G14-GA401-Hackintosh/tree/Ventura.

**Comment by dakinianaya** [comment]
Not only on Ventura. I tried Sonoma as well. Wifi network not showing in recovery. Usb tethering also not working. I checked for kextstat, it is loaded. Don't know what's going wrong. All other things working like trackpad, usb mouse. 
![IMG20231209110123](https://github.com/PIut02/ROG-Zephyrus-G14-GA401-Hackintosh/assets/67649680/e6e8f027-91d4-4a46-b48c-9a2fed94c528)

**Comment by PIut02** [comment]
I haven't tried to install it in recovery mode so I don't know what's going on, maybe you should use a USB flash image to install.

**Comment by dakinianaya** [comment]
Oh!! Unfortunately none of my friends have macbook to create offline usb installer. My dreams ends here then. Thanks @PIut02 for all the help.

**Comment by PIut02** [comment]
Well, you can download the image from some websites, and then flash it to a USB drive on Windows.

**Comment by dakinianaya** [comment]
I read somewhere that we can't create offline installer in windows post Catalina10.15. I may not be aware but is it possible to create Sonoma/Ventura in Windows? Direct me if there is something.

**Comment by PIut02** [comment]
Download the image from: https://github.com/corpnewt/gibMacOS
Flash USB: http://www.etcher.io/

**Comment by dakinianaya** [comment]
I will give it a try. Thanks

**Comment by dakinianaya** [comment]
Hi @PIut02 Somehow I managed to install Ventura using usb installer. Thanks for the tip. As you can see from the below image, I have graphics display only 7 mb. I allocated 2GB in UMAF app. but still it is showing only 7 mb. any idea? UI is kinda of laggy.
![Screenshot 2023-12-09 at 3 54 21 PM](https://github.com/PIut02/ROG-Zephyrus-G14-GA401-Hackintosh/assets/67649680/04592a04-f88d-4aa4-9461-fad6148023ab)

**Comment by PIut02** [comment]
Just add `NootedRed` in the kext.

**Comment by dakinianaya** [comment]
![Screenshot 2023-12-09 at 4 55 06 PM](https://github.com/PIut02/ROG-Zephyrus-G14-GA401-Hackintosh/assets/67649680/4b364a12-b6c9-4061-bac7-79886e5e54aa)
Unfortunately I have it already(added after installation) but doesn't seem to be working,

**Comment by dakinianaya** [comment]
Hey @PIut02 I am done finally. :) I reinstalled Sonoma then had issues with NootedRed - black screen issue while booting. Resolved by changing the Model to MacbookPro16,3. All seems working now. Registering it here so that someone may find it useful. Thank you so much for all the help. Keep up the good work. 
![Screenshot 2023-12-09 at 20 53 23](https://github.com/PIut02/ROG-Zephyrus-G14-GA401-Hackintosh/assets/67649680/8c5a4a54-388d-490a-b29b-1c78ede8dd21)
